src/schema3.iced

```diff
@@ -165,8 +165,13 @@ class ChainType extends Node
     return null
 
 class String extends Node
+  constructor : ({args}) ->
+    @_args = args
+
   _check : ({path, obj}) ->
     if typeof(obj) isnt 'string' or obj.length is 0 then return mkerr path, "value must be a string"
+    if @_args? and @_args.maxLength?
```

Comment:
use snake_case in iced

Reply:
if you like, you can do `if (max_length = @_args?.max_length)?` as well, then use that value in the next block